tests/test_scene_rendering/test_cli_flags.py

```diff
@@ -91,6 +91,30 @@ def test_s_flag_no_animations(tmp_path, manim_cfg_file, simple_scenes_path):
     assert not is_empty, "running manim with -s flag did not render an image"
 
 
+@pytest.mark.slow
+def test_no_s_flag_text(tmp_path, manim_cfg_file, simple_scenes_path):
+    scene_name = "Text"
+    command = [
+        "python",
+        "-m",
+        "manim",
+        simple_scenes_path,
+        scene_name,
+        "-ql",
+        "--media_dir",
+        str(tmp_path),
+    ]
+    out, err, exit_code = capture(command)
+    assert exit_code == 0, err
+
+    is_empty = not any((tmp_path / "videos" / "simple_scenes" /
+                        "480p15" / "partial_movie_files" / scene_name).iterdir())
+    assert is_empty, "running manim on text scene without -s flag rendered a video"
+
+    is_empty = not any((tmp_path / "images" / "simple_scenes").iterdir())
+    assert not is_empty, "running manim on scene with no animation without -s flag did not render an image"
```

Comment:
keep messages consistent. Ie. both should say "running manim on scene with no animation without -s flag...."